app/assets/components/phase-banner/_index.scss

```diff
@@ -0,0 +1,27 @@
+@import "../tag/index";
```

Comment:
@goodvibes - Is this a duplicate of `app/assets/components/phase-banner/_phase-banner.scss` ? If so, could we get rid of it ?

Reply:
It's not, the tag .scss handles the CSS around the Alpha / Beta tag in the original phase banner. The tag class in _phase-banner.scss is just tweaking a much larger class.

Reply:
Thanks @goodvibes for the commit 883c1d26296b133da756d71755ff2d712a6cbb1c to remove `app/assets/components/phase-banner/_index.scss` as `tag` component specific CSS classes have already been included as part of `nhsuk-frontend` (as discussed) 👍